AP_Scripting: add binding for ahrs::get_variances

rmackay9 opened the pull request:
This adds a binding for lua scripting so that the EKF's variances can be retrieved.  A project I'm working on required this so that a script controlling the vehicle in Guided mode could pause the movement of the vehicle if it seemed to be losing control.

This has been tested in SITL and will likely be tested on a real vehicle tomorrow.

Below is a screen shot (from SITL) of the variances being printed.  I tested that the position and compass variances changed by modifying SIM_GPS_GLITCH_X and SIM_MAG_SCALING parameters.
![scripting-ahrs-binding-tests](https://user-images.githubusercontent.com/1498098/96114586-c5647f80-0f20-11eb-875a-0f4bc66bb79a.png)

rmackay9 commented:
This has been successfully tested on a real-vehicle now.

tridge commented on libraries/AP_Scripting/examples/ahrs-print-variances.lua:
```diff
@@ -0,0 +1,13 @@
+-- This script displays the ahrs variances at 1hz
+
+function update() -- this is the loop which periodically runs)
+  vel_variance, pos_variance, height_variance, mag_variance, airspeed_variance, offset = ahrs:get_variances()
```
let's remove the offset

rmackay9 replied:
txs for the review.  I've removed the offset in the follow-up commits.

rmackay9 commented:
As discussed on the dev call, I've removed the unused "offset" argument from the get_variances() call, rebased on master and re-tested that the original functionality of retrieving variances via scripting still works.  I've also test that Copter's EKF failsafe still works.

rmackay9 commented:
@tridge, @peterbarker If there are no more comments I'll merge after this passes travis.
